AgentQnA/tests/1_build_images.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+WORKPATH=$(dirname "$PWD")
+export WORKDIR=$WORKPATH/../../
+echo "WORKDIR=${WORKDIR}"
+export ip_address=$(hostname -I | awk '{print $1}')
+
+
+function get_genai_comps() {
+    cd $WORKDIR
+    if [ ! -d "GenAIComps" ] ; then
+        git clone https://github.com/opea-project/GenAIComps.git
+    fi
+}
+
+function build_docker_images_for_retrieval_tool(){
+    cd $WORKDIR/GenAIComps/
+    echo $PWD
+    echo "==============Building dataprep image================="
```

Comment:
Please create a similar compose yaml for image build, https://github.com/opea-project/GenAIExamples/blob/main/AudioQnA/docker/docker_build_compose.yaml, under `AgentQnA/docker`, this will accelerate image build by parallel build, and it will be used for image publication as well.

Reply:
@chensuyue ok. Just want to clarify: I just need to create such a file, but not need to actually use that docker build file to build images in the unit test, right? because some images are not published yet.